### DISCUSSION TITLE
Option to start new ingestion with depth 1

### DISCUSSION THREAD

**Discussion by gammazero** [filecoin-project/storetheindex#1008] [Ideas]
To start ingesting only content published from the time an indexer is started, a new config option could be added to make the indexer behave this way.

For any new provider, only ingest the head of the ad chain (depth 1).

Is there a use case for this?

**Comment by masih** [comment]
Please update the description with the motivation for the idea. What problem does the idea solve?
Is this potentially a duplicate of https://github.com/filecoin-project/storetheindex/discussions/447 ?
